examples/interfaces/cw1/src/lib.rs

```diff
@@ -11,35 +11,43 @@ pub struct CanExecuteResp {
 }
 
 #[interface]
-#[sv::custom(msg=cosmwasm_std::Empty, query=cosmwasm_std::Empty)]
 pub trait Cw1 {
     type Error: From<StdError>;
+    type ExecC: CustomMsg;
+    type QueryC: CustomQuery;
+    type CosmosCustomMsg: CustomMsg;
 
     /// Execute requests the contract to re-dispatch all these messages with the
     /// contract's address as sender. Every implementation has it's own logic to
     /// determine in
     #[sv::msg(exec)]
-    fn execute(&self, ctx: ExecCtx, msgs: Vec<CosmosMsg>) -> Result<Response, Self::Error>;
+    fn execute(
+        &self,
+        ctx: ExecCtx<Self::QueryC>,
+        msgs: Vec<CosmosMsg<Self::CosmosCustomMsg>>,
```

Comment:
Didn't we want to make it

```suggestion
        msgs: Vec<CosmosMsg<Self::ExecC>>,
```

Reply:
If I change this line I get `associated type ExecC not found for Self`.

And also the `#[interface]` directive errors with:

```
type alias takes 1 generic argument but 0 generic arguments were supplied
expected 1 generic argumentrustc
lib.rs(13, 1): type alias defined here, with 1 generic parameter: `ExecC`
lib.rs(13, 2): add missing generic argument: `ExecC`
```

Do I need to use the sv::custom directive?

Reply:
Ok, I thought the [fix](https://github.com/CosmWasm/sylvia/pull/406) landed in this PR.

Leave it as it is and remove `CosmosCustomMsg` in following PR then.